en/drafts/originals/analyzing-multilingual-text-nltk-spacy-stanza.md

```diff
@@ -24,91 +24,99 @@ doi: XX.XXXXX/phen0000
 
-- Part-of-speech (POS) tagging
-  - POS tagging involves marking each word of a text with its corresponding part-of-speech (e.g. nouns, verbs, adjectives, etc.). POS taggers can use rule-based algorithms, following set definitions to infer the proper tag to attach to a word, or stochastic (probabilistic) approaches, in which the tagger calculates the probability of a given set of tags occurring and assigns them to an untrained text. For example, a sentence with parts of speech tagged by NLTK might look like the following (this sentence is taken from Chapter 5 of the [NLTK Book](https://www.nltk.org/book/ch05.html)): [('And', 'CC'), ('now', 'RB'), ('for', 'IN'), ('something', 'NN'), ('completely', 'RB'), ('different', 'JJ')]. The sentence is split into its component words, and each word is put into a tuple with its corresponding POS tag.
+In this lesson, we will start by tokenizing our text. This will enable to us to then perform POS tagging and lemmatization on the textual data. Without prior tokenization, we would not be able to access the text as a series of words, to which we can apply our tagging and lemmatization.
 
-{% include figure.html filename="en-or-analyzing-multilingual-text-nltk-spacy-stanza-01.png" alt="A screenshot of Python code showing a sample sentence being tagged into a list of tuples containing its component words and their respective parts of speech using NLTK." caption="Screenshot of part-of-speech tagging from the NLTK Book, Chapter 5." %}
+**Part-of-Speech (POS) tagging**
 
-- Lemmatization
-  - Lemmatization reduces a word to its dictionary form, which is known as a "lemma." The lemmatized form of the word "coding," for example, would be "code," with the "-ing" suffix removed.
+POS tagging involves marking each word in a text with its corresponding part-of-speech (e.g. 'noun', 'verb', 'adjective', etc.). POS taggers can use rule-based algorithms, following set definitions to infer the proper tag to attach to a word, or [stochastic](https://en.wikipedia.org/wiki/Stochastic) (probabilistic) approaches, in which the tagger calculates the probability of a given set of tags occurring, and assigns them to an untrained text. The sentence used as an example for tokenization above, for example, might look like the following with its parts of speech tagged by NLTK: [('And', 'CC'), ('now', 'RB'), ('for', 'IN'), ('something', 'NN'), ('completely', 'RB'), ('different', 'JJ')]. The sentence is split into its component words, and each word is put into a [tuple](https://en.wikipedia.org/wiki/Tuple) with its corresponding POS tag.
```

Comment:
As you can see here with 'scholastic' and 'tuple', I've inserted some hyperlinks to Wikipedia articles to define some of the more uncommon/technical language used in the lesson. This is a common practice in our Programming Historian publications, but please reach out if you would prefer to remove or replace these links with alternatives.